.bumpversion.cfg

```diff
@@ -0,0 +1,6 @@
+[bumpversion]
```

Comment:
Maybe add a comment to this file with a pointer to the bumpversion tool?

Reply:
Tried that, but it appears that bumpversion removes all comments from the file after you execute it. So we should add more details to the contributing file when we have one (i.e., a link to the tool and the process we follow to tag new versions).